src/v/rpc/server.cc

```diff
@@ -131,8 +136,8 @@ ss::future<> server::stop() {
     ss::sstring proto_name = _proto ? _proto->name() : "protocol not set";
     vlog(
       rpclog.info, "{} - Stopping {} listeners", proto_name, _listeners.size());
-    for (auto&& l : _listeners) {
```

Comment:
sounds like `&&` was a bug?

Reply:
no i don't think it is a bug. it's just the value type is known and it looks like there nothing to std::forward